operator/src/main/java/oracle/kubernetes/operator/ServerStatusReader.java

```diff
@@ -167,6 +168,8 @@ public NextAction apply(Packet packet) {
             String state = null;
             ClientPool helper = ClientPool.getInstance();
             ApiClient client = helper.take();
+            String namespace = pod.getMetadata().getNamespace();
+            LoggingContext.context(new LoggingContext().namespace(namespace));
```

Comment:
Does this need a corresponding LoggingContext().remove() in a finally?

Reply:
fixed